### ISSUE TITLE
Add command to upload custom manifest if first upload failed

### ISSUE THREAD

**Issue by fvilers** [appget/appget#18]
Hi! I created a custom manifest but for some reason the upload to the official repository has failed.

```
Couldn't submit manifest Request to https://octobot.appget.net/pr has timed out.
```

It could be useful to add a `submit` or `upload` command to retry the submission.

**Comment by fvilers** [comment]
Strange, it looks like the manifest has been uploaded as there's a PR for it (https://github.com/appget/appget.packages/pull/2750).

**Comment by kayone** [comment]
yeah, the request might timeout (we might not be able to send you a response back in time) but it would be rare for it to fail. I've marked this as an enhancement since it is a nice to have but not a priority right now.
